Sources/Networking/Operations/Handling/CustomerInfoResponseHandler.swift

```diff
@@ -17,11 +17,26 @@ class CustomerInfoResponseHandler {
 
     private let offlineCreator: OfflineCustomerInfoCreator?
     private let userID: String
+    private let failIfInvalidSubscriptionKeyDetectedInDebug: Bool
+    private let isDebug: () -> Bool
```

Comment:
should this store the boolean directly instead of the lambda?

Reply:
Yeah, I've made this a bool and switched the constructor argument to use the autocompleting lambda!